### PR TITLE
SASS. @each. Replace stub

### PR DESCRIPTION
**Pull request by TylerGlenski**
Added 1x guide/example, 1x problem, and 1x solution.

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [ x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: guide/english/certifications/front-end-libraries/sass/use-each-to-map-over-items-in-a-list/index.md [hunk 1]
```diff
@@ -3,8 +3,57 @@ title: Use @each to Map Over Items in a List
...
+    color: $color;
+  }
+}
+<div class="white-font"></div>
+<div class="black-font"></div>
+<div class="blue-font"></div>
+```
+---
+
+## Problem:
```
Comment (thecodingaviator) [inline guide/english/certifications/front-end-libraries/sass/use-each-to-map-over-items-in-a-list/index.md]:
```suggestion

```
You do not need to restate the problem here, just put the hints and solution

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Remove the whole problem section please, the user can go back to the curriculum if they want to see the problem again

**Comment by thecodingaviator** [comment]
I see that there are improvements made by this PR, however, #34792 made changes to the same file adding the solution. Since it was a previous PR, it was merged first.

If you could please fix the merge conflict and update your file to what you want to add, please do. If you want to close the PR, that's fine too.

Thank you for contributing! Happy coding! ✈️

**Comment by RandellDawson** [comment]
@thecodingaviator I fixed the merge conflict on behalf of the user, so the text and example added by the OP is included in this article.

If you are fine with the changes, go ahead and merge it.

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by thecodingaviator** [comment]
@TylerGlenski Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**